### ISSUE TITLE
breadcrumb trail : feedback & problem

### ISSUE THREAD

**Issue by planetf1** [odpi/egeria#1958]
Scenario
 * Rebuild, launch UI
 * login as faithbroker

1. Initial location is shown as 'Home->Asset Search' - this is a little confusing as I'd expect to land on home, though since the asset search 'tab' is selected this makes some sense (but feels confusing)
2. Click on 'Asset Lineage'. Breadcrumb now shows as 'Home -> Asset Lineage -> ultimate Source ->' - the extraneous '->' at the end seems unnecessary and again the inclusion of 'ultimate source' seems confusing as navigation is now a mix of our vertical 'tabs' and horizontal tabs. 
3. Click on 'Asset lineage' in the breadcrumb. I now end up at 'Home > Asset Lineage >' This is same 'location' navigation as in 2/ yet displays differently - with no 'Ultimate source' despite this being highlighted
4. I can click on 'subject area' which shows up ok (other than general confusion)
5. Type explorer is ok too
6. I then go back to Asset Lineage (Showing Home > Asset Lineage > ultimateSource >' then click on 'Subject Area'. The breadcrumb now shows as 'Home > Subject Area > Ultimate Source >'. 

So in summary
 * The logical model here seems confusing given our mix of vertical and horizontal navigation. The breadcrumb trail makes it seem more confusing not less'
 * I don't think the trail should end in > (minor)
 * There are specific issues going to /from lineage with the third element being left in the trail

**Comment by ruxandraRosu** [comment]
Hi Nigel,
I will look into this issue. Regarding the horizontal navigation being in the breadcrumb, we can discuss about it in the next functionality call.

**Comment by ruxandraRosu** [comment]
@planetf1 @grahamwallis @davidradl 
Hi,
Would you please check PR https://github.com/odpi/egeria/pull/1995 for this issue?

**Comment by planetf1** [comment]
I suggest we discuss the broader aspects of UI design & architecture at our next face to face meeting. Some of the code defects are addressed, but I suggest we leave this open and target for 1.3 to consider usability/approach

**Comment by planetf1** [comment]
This was feedback but the UI is now evolving with feedback by the dev team so will close this now
